core/sds-aspect-meta-model-java/src/test/java/io/openmanufacturing/sds/metamodel/loader/AspectMetaModelInstantiatorTest.java

```diff
@@ -43,7 +43,9 @@
 
 public class AspectMetaModelInstantiatorTest extends MetaModelInstantiatorTest {
    @ParameterizedTest
-   @EnumSource( value = TestAspect.class )
+   @EnumSource( value = TestAspect.class, mode = EnumSource.Mode.EXCLUDE, names = {
+         "ASPECT_WITH_TIME_SERIES" // This feature branch does not support bamm:AbstractProperty. When support for this is added, remove this
```

Comment:
The PR for the `AbstractProperty` is merged, right? If so couldn't the main branch be merged into this branch and this exclusion resolved?

Reply:
No, actually the feature branch for AbstractProperty is based on this branch here. It is already reverted in this branch: https://github.com/bci-oss/sds-sdk/commit/023c451be48ff013ee489f1814a0f0a22d46fc28